lib/constants.dart

```diff
@@ -5,7 +5,7 @@ class EnboxConstants extends InheritedWidget {
 
   final String displayName = "enbox";
   final Color color = Colors.blue;
-  final String demoName = "Enbox Demo";
+  final String demoName = "Enbox Demo 2.0";
```

Comment:
Is this appropriate?

Reply:
this is just demo data anyways, and i just changed this to indicate a successfull automated build at mail.enbox.app, so we know it updated

Reply:
Sure, since it's on web.

Reply:
Got it. This is a nitpick at this point, but maybe change this to 0.0.1 to signify a "minor" change